client/assets/stylesheets/_p2-vars.scss

```diff
@@ -17,6 +17,8 @@
 	--p2-color-button: #f3f3f3;
 	--p2-color-button-dark: #eaeaea;
 
+	--p2-color-background-light: #f3f3f3;
```

Comment:
Seems like we already have these colors defined just above these definitions. Should we use those variables or do you think it's better if we duplicate them but with more suitable naming like you did?

Reply:
I did that on purpose as there have been times that Edu/David tweaked the color values. Separate naming for separate elements would avoid confusion there.